Analyzer/src/MakeMiniTree.cc

```diff
@@ -10,110 +10,195 @@ MakeMiniTree::MakeMiniTree()
+            variables.insert("GenParticles_PdgId");
+            variables.insert("GenParticles_ParentId");
+            variables.insert("GenParticles_ParentIdx");
+            variables.insert("GenParticles_Status");
+            variables.insert("ScaleWeights");
+            variables.insert("PSweights");
+            variables.insert("PDFweights");
+            variables.insert("puWeight");
+            variables.insert("puSysUp");
+            variables.insert("puSysDown");
+            variables.insert("NonPrefiringProb");
+            variables.insert("NonPrefiringProbUp");
+            variables.insert("NonPrefiringProbDown");
+            variables.insert("Jets_origIndex");
+            variables.insert("JetsJECup_origIndex");
+            variables.insert("JetsJECdown_origIndex");
+            variables.insert("JetsJERup_origIndex");
+            variables.insert("JetsJERdown_origIndex");
+            variables.insert("Jets_jerFactor");
+            variables.insert("Jets_jecUnc");
+            variables.insert("JetsJECup_jerFactor");
+            variables.insert("JetsJECdown_jerFactor");
+            variables.insert("Jets_jerFactorUp");
+            variables.insert("Jets_jerFactorDown");
+            variables.insert("JetIDJECdown");
+            variables.insert("JetIDJECup");
+            variables.insert("JetIDJERdown");
+            variables.insert("JetIDJERup");
+            variables.insert("JetsAK8_origIndex");
+            variables.insert("JetsAK8JECup_origIndex");
+            variables.insert("JetsAK8JECdown_origIndex");
+            variables.insert("JetsAK8JERup_origIndex");
+            variables.insert("JetsAK8JERdown_origIndex");
+            variables.insert("JetsAK8_jerFactor");
+            variables.insert("JetsAK8_jecUnc");
+            variables.insert("JetsAK8JECup_jerFactor");
+            variables.insert("JetsAK8JECdown_jerFactor");
+            variables.insert("JetsAK8_jerFactorUp");
+            variables.insert("JetsAK8_jerFactorDown");
         }
+
         if( tr.isFirstEvent() ) {
-            std::string myTreeName = "myMiniTree";
+            std::string myTreeName = "SkimmedTree";
             myTree = new TTree( (myTreeName).c_str() , (myTreeName).c_str() );
             myMiniTuple = new MiniTupleMaker( myTree );
             myMiniTuple->setTupleVars(variables);
             myMiniTuple->initBranches(tr);
         }
         
-        if( runtype == "MC" ) {
-            const auto& passTriggerMC       = tr.getVar<bool>("passTriggerMC");
-            const auto& passMadHT           = tr.getVar<bool>("passMadHT");
-            
-            if( !passMadHT || !passTriggerMC ) continue; 
-        }
-
         //-----------------------------------
         //-- Fill Histograms Below
         //-----------------------------------
-        if( passBaseline1l ) {
+        if( NGoodJets_pt30 >= 5 and HT_trigger_pt30 > 400.0 ) {
+            eventCounter->Fill(-1.0);
```

Comment:
I'm confused by filling -1 here now. lol.

Reply:
Oh yeah, #codeabuse

Reply:
Resolved in #65208e0